boot/bootdata/CMakeLists.txt

```diff
@@ -103,7 +103,7 @@ add_livecd_shortcut("Notepad" notepad.exe "Profiles/All Users/Start Menu/Program
 add_livecd_shortcut("WordPad" wordpad.exe "Profiles/All Users/Start Menu/Programs/Accessories")
 add_livecd_shortcut("Solitaire" sol.exe "Profiles/All Users/Start Menu/Programs/Games")
 add_livecd_shortcut("Spider Solitaire" spider.exe "Profiles/All Users/Start Menu/Programs/Games")
-add_livecd_shortcut("WineMine" winmine.exe "Profiles/All Users/Start Menu/Programs/Games")
+add_livecd_shortcut("Minesweeper" winmine.exe "Profiles/All Users/Start Menu/Programs/Games")
```

Comment:
If you limit the changes to only renaming the shortcuts, I think this could be acceptable.